lib/ddtrace/contrib/active_support/notifications/subscriber.rb

```diff
@@ -21,7 +21,12 @@ def subscriptions
 
             # Returns whether subscriptions have been activated, via #subscribe!
             def subscribed?
-              subscribed == true
```

Comment:
The original reason for doing a `subscribed == true` instead of `!subscriptions.empty?` was because this subscription process was conceived as one-way thing, much like our patchers. Maybe this is okay, but it suggests to me a change in strategy, which should be considered thoughtfully.

Reply:
Agreed, I think I've made this change before implementing `unsubscribe_all` and with proper use of that this change should no longer be needed.
Thanks for pointing that out.